web/ulisp.js

```diff
@@ -1715,11 +1716,11 @@ var wasmImports = {
 var wasmExports = await createWasm();
 var ___wasm_call_ctors = createExportWrapper('__wasm_call_ctors', 0);
 var _free = Module['_free'] = createExportWrapper('free', 1);
+var _fflush = createExportWrapper('fflush', 1);
 var _print_version = Module['_print_version'] = createExportWrapper('print_version', 0);
 var _setup = Module['_setup'] = createExportWrapper('setup', 0);
 var _stop_loop = Module['_stop_loop'] = createExportWrapper('stop_loop', 0);
 var _evaluate = Module['_evaluate'] = createExportWrapper('evaluate', 1);
-var _fflush = createExportWrapper('fflush', 1);
```

Comment:
Ah, so `_fflush` was exported to JS host side by default.

Anyway, all the changes in `web/ulisp.js` and `public/ulisp.wasm` can be reverted. They'll get regenerated on `bun build:web`.